amundsen_application/static/js/ducks/search/reducer.ts

```diff
@@ -49,13 +49,14 @@ export interface SearchReducerState {
 };
 
 /* ACTIONS */
-export function searchAll(term: string, resource?: ResourceType, pageIndex?: number, useFilters: boolean = false): SearchAllRequest {
+export function searchAll(searchType: string, term: string, resource?: ResourceType, pageIndex?: number, useFilters: boolean = false): SearchAllRequest {
```

Comment:
nit: `searchType` is the first reducer param but the last API param. Actually all of the fields are in a different order.

Would like to see some consistency, or convert ordered params into a map.

Reply:
I'd like to do this with future work consolidating sagas, and yes this just follows existing tech debt where there are all out of order. I prefer maps moving forward, and maybe we can start that with the dashboard work and I can try to clean this up on existing code on `master` in parallel.